client/components/clubs/Search.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components'
```

Comment:
I was confused initially with the filename and what the contents contianed.

Suggestion: `SearchInput.tx`, `SearchBar.tsx`...